src/edit/dompos.js

```diff
@@ -258,7 +258,7 @@ function targetKludge(dom, coords) {
 
 // Given an x,y position on the editor, get the position in the document.
 function posAtCoords(pm, coords) {
-  let elt = targetKludge(document.elementFromPoint(coords.left, coords.top + 1), coords)
+  let elt = targetKludge(pm.root.elementFromPoint(coords.left, coords.top + 1), coords)
```

Comment:
Can this code be executed when `pm.root` is `null`? If so a null check may be worthwhile.